app/controllers/graphql_controller.rb

```diff
@@ -1,20 +1,45 @@
 class GraphqlController < ApplicationController
   skip_before_action :verify_authenticity_token
 
-  rescue_from 'ActiveRecord::RecordInvalid' do |exception|
-    exception.record.errors.messages
-    render json: {
-      errors: exception.record.errors,
+  def execute
+    variables = ensure_hash(params[:variables])
+    context = {
+      current_user: current_user
     }
+
+    render json: PortalSchema.execute(params[:query],
+      variables: variables,
+      context: context,
+      operation_name: params[:operationName])
```

Comment:
why is `:operationName` not snake case at this point?

Reply:
I believe this is before the query has been processed and the conversion has happened. We are in a controller here, not yet in the graphql resolution portion. This controller was auto generated by the new gem.

Reply:
i thought so but didn't know if we could move the camelCase -> snake_case step earlier via a library function

Reply:
I'm not aware of them exposing a nice function for it. I mean, we have activesupport so we could call `underscore` if we wanted to, but that would involve manipulating the hash. I'm just not sure if its worth manipulating the data before forwarding it into the graphql execution layer.